tutorials/units-and-integration/units-and-integration.ipynb

```diff
@@ -0,0 +1,234 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fbf1f550b38>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "nu = np.linspace(1., 3000., 1000) * u.THz\n",
+    "bb5000K_nu = blackbody_nu(in_x=nu, temperature=5000. * u.Kelvin)\n",
+    "plt.plot(nu, bb5000K_nu)\n",
+    "plt.xlabel(r'$\\nu ~ [{\\rm THz}]$')\n",
+    "plt.ylabel(r'$I_{\\nu} ~ [{\\rm erg/s/cm^2/Hz/sr}]$')\n",
+    "plt.title('Planck function in frequency')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "WARNING: UnitsWarning: 'erg/s/cm2/sr' contains multiple slashes, which is discouraged by the FITS standard [astropy.units.format.generic]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/latex": [
+       "$1.1280834 \\times 10^{10} \\; \\mathrm{\\frac{erg}{s\\,sr\\,cm^{2}}}$"
+      ],
+      "text/plain": [
+       "<Quantity 11280833528.711555 erg / (cm2 s sr)>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "np.trapz(x=nu, y=bb5000K_nu).to('erg/s/cm2/sr')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can do something similar, but for a wavelength grid. We want to integrate over an equivalent wavelength range to the frequency range we did earlier. We can transform the maximum frequency into the corresponding (minimum) wavelength by using the `.to()` method, with the addition of an *equivalency*."
```

Comment:
Maybe mention how to adjust the plot limits for this one (since the default limits squish the interesting parts near the origin).